core/lib/common/test.js

```diff
@@ -43,6 +43,25 @@ module.exports = class Test {
 
 		this.archiver = Archiver;
 		this.context = new Context(this.suite.context);
+
+		// Return a proxy of this object that will forward access to properties
+		// present in the context object to that object.
+		//
+		// This allows us to shorten, e.g.:
+		//
+		// this.context.get().worker.executeCommandInHostOS(...)
+		//
+		// To just:
+		//
+		// this.worker.executeCommandInHostOS(...)
```

Comment:
```suggestion
		/** Return a proxy of this object that will forward access to properties
		* present in the context object to that object.
		*
		* This allows us to shorten, e.g.:
		*
		* this.context.get().worker.executeCommandInHostOS(...)
		*
		* To just:
		*
		* this.worker.executeCommandInHostOS(...)
    */
```

Reply:
JSdoc styled comments are parseable and can be picked up by typedoc

Reply:
Ah, good suggestion